sbin/prepareWorkspace.sh

```diff
@@ -817,6 +820,18 @@ downloadBootJdkIfNeeded () {
   fi
 }
 
+downloadGtest() {
+  local gtestUnpacked="${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}/installedgtest"
+  if [ -e "${gtestUnpacked}" ]; then
+    echo "Reusing $gtestUnpacked"
+  else
```

Comment:
Again just idea to keep build able to work offline -  eg to skip the gtrests if the binary is not available ale and some --no-gtests present on cmdline? Check/download otherwise.... WDYT?

Reply:
yes, maybe something like `--skip-gtest` as there are -`-skip-alsa` `--skip-freetype` [switches](https://github.com/adoptium/temurin-build/blob/d017e2c38ecf8117ba62471652e069b67376e19c/sbin/common/config_init.sh#L314).